src/core_plugins/kibana/public/dashboard/dashboard_constants.js

```diff
@@ -2,7 +2,10 @@
 export const DashboardConstants = {
   ADD_VISUALIZATION_TO_DASHBOARD_MODE_PARAM: 'addToDashboard',
   NEW_VISUALIZATION_ID_PARAM: 'addVisualization',
-  LANDING_PAGE_PATH: '/dashboard',
-  CREATE_NEW_DASHBOARD_URL: '/dashboard/create',
-  EXISTING_DASHBOARD_URL: '/dashboard/{{id}}'
+  LANDING_PAGE_PATH: '/dashboard/list',
+  // NOTE, the following two urls have to have LANDING_PAGE_PATH as their sub url for
+  // the chrome nav links to work correctly (eg. so navigating away from an opened dashboard, and back to
+  // the dashboard app, will reopen the dashboard, not show the landing page.
```

Comment:
Or maybe I'm misunderstanding this entirely. Why can't we use `dashboard/create` and `dashboard/edit`? If we handle those routes before handling `dashboard/:id`, then they won't be interpreted as IDs.